rnndb/state.xml

```diff
@@ -86,11 +86,14 @@ xsi:schemaLocation="http://nouveau.freedesktop.org/ rules-ng.xsd">
         <value value="3" name="UNSIGNED_SHORT"/>
         <value value="4" name="INT"/>
         <value value="5" name="UNSIGNED_INT"/>
+        <value value="6" name="INT_10_10_10_2"/>
```

Comment:
I think these values have the following meaning (at least gc3000).
6: INT_2_10_10_10_REV
7: UNSIGNED_INT_2_10_10_10_REV

With that you should leave 12 and 13 untouched.

Reply:
What does the REV mean in this case?

Reply:
reversed - regarding how the packed data is stored.

Reply:
I see, this matches the names of the types accepted by `glVertexAttribPointer` in GLES3. Changed it.